pom.xml

```diff
@@ -190,6 +190,16 @@
                 <artifactId>maven-plugin-annotations</artifactId>
                 <version>${maven-plugin-tools.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-core</artifactId>
+                <version>5.14.2</version>
+            </dependency>
+            <dependency>
+                <groupId>org.junit-pioneer</groupId>
+                <artifactId>junit-pioneer</artifactId>
+                <version>2.3.0</version>
+            </dependency>
```

Comment:
Should these be `<scope>test</scope>`?

Reply:
They're just managed dependencies in the root POM. They have the correct scope in the child projects.